src/Adapter/PlatformServices.Desktop/PlatformServices.Desktop.csproj

```diff
@@ -116,14 +134,53 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <None Include="..\PlatformServices.Shared\netstandard1.3\Resources\xlf\Resource.cs.xlf">
```

Comment:
This seems to be a change from the earlier model.. any reason..?

Reply:
We are keeping the resources file and language xlf's in shared folder as these are common for netcoreapp and desktop. Adding them as links in desktop and netcore creates this lines in csproj.